content/en/containers/kubernetes/installation.md

```diff
@@ -177,7 +177,7 @@ agent:
         - <GROUP_ID>
 {{< /highlight >}}
 
-- Replace `<USER_ID>` with the UID to run the Datadog Agent.
+- Replace `<USER_ID>` with the UID to run the Datadog Agent. [Datadog recommends to set 100 since Datadog Agent v7.48+][26].
```

Comment:
```suggestion
- Replace `<USER_ID>` with the UID to run the Datadog Agent. [Datadog recommends setting this value to 100 since Datadog Agent v7.48+][26].
```

Reply:
Could we provide some kind of brief explanation for this here?

Reply:
Only a placeholder doesn't indicate what value to set. Since the documentation provides a recommended value, I'd like to include a link to it here.

https://github.com/DataDog/documentation/blob/63b3271a3abda3ba035d3773fba3dc07ec97c344/content/en/data_security/kubernetes.md?plain=1#L50-L51

Reply:
Ah ok, I'll update my suggestion. By the way, this content is actually in a code block, so links need to be the complete URLs.